seatunnel-dist/release-docs/LICENSE

```diff
@@ -220,7 +220,8 @@ The text of each license is the standard Apache 2.0 license.
      (Apache License 2.0) aircompressor (io.airlift:aircompressor:0.10 - http://github.com/airlift/aircompressor)
      (Apache License, Version 2.0) Apache Yetus - Audience Annotations (org.apache.yetus:audience-annotations:0.11.0 - https://yetus.apache.org/audience-annotations)
      (The Apache Software License, Version 2.0) Apache Avro (org.apache.avro:avro:1.8.2 - http://avro.apache.org)
-     (Apache License, Version 2.0) Apache Commons Codec (commons-codec:commons-codec:1.13 - https://commons.apache.org/proper/commons-codec/)    
+     (The Apache Software License, Version 2.0) Apache Avro (org.apache.avro:avro:1.11.1 - http://avro.apache.org)
```

Comment:
Can we use one Avro version here? I find there is already exist `1.8.2`

Reply:
I use some feature released after avro version 1.8.2 (I reminber it's 1.9.x), To handle timestamp with java Instant rather than joda time.

Reply:
So it might be better to remove `org.apache.avro:avro:1.8.2`